Core/Object Arts/Dolphin/Base/PackageManager.cls

```diff
@@ -510,7 +510,10 @@ install: aString
 	[newPackages do: 
 			[:each | 
 			self addPackage: each.
-			[self basicInstall: each] ifCurtailed: [self basicUninstall: each].
+			[
+				self basicInstall: each.
+				self trigger: #packageInstalled with: each.
```

Comment:
The event selector should be #packageInstalled: (i.e. with a terminating colon) because it has an argument. Although this is mainly a convention, it is standard practice for reasons of readability.